base/db/R/query.prior.R

```diff
@@ -31,7 +32,7 @@
 ##'   con <- db.open(...)
 ##'   query.priors("ebifarm.pavi", c("SLA", "Vcmax", "leaf_width"), con = con)
 ##' }
-query.priors <- function(pft, trstr = NULL, con = NULL, ...){
+query.priors <- function(pft, trstr = NULL, con = NULL, update = FALSE, ...){
```

Comment:
Why is `update` being added to query.priors? I don't see anywhere this variable is used within the function, nor does the concept of `update` make sense in a function that only does a SELECT but no INSERT